onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -368,23 +368,21 @@ struct CpuBuffersInfo {
   // CUDA stream. For example, this fields
   // should contain all values in
   // deferred_release_buffer_pool_[my_stream]
-  // when release my_stream's buffers.
-  void** buffers;
-  // CPU buffer buffers[i].
-  // Number of buffer points in "buffers".
-  size_t n_buffers;
+  // when release my_stream's (type: cudaStream_t)
+  // buffers.
+  std::vector<void*> buffers;
```

Comment:
In the future, please avoid using raw pointers. You should use unique_ptr or shared_ptr instead. For example, if this is a unique_ptr, then in ReleaseCpuBufferCallback you can release the unique_ptr and free the pointer. And if someone forgot to call ReleaseCpuBufferCallback, the memory will be recycled when this EP is destructed.

Reply:
This is a value returned by other CUDA EP APIs. I don't change its type. It stands for memory allocated on CPU and could be `float*`, `int*`, etc, so I think it needs to be a `void*`.